website/docs/components/table/partials/guidelines/guidelines.md

```diff
@@ -155,4 +155,47 @@ Row placement determines the visual styling based on where the row is placed rel
 - We recommend using `medium` cell density by default.
 - If content is complex or a smaller data set (e.g., a Table of basic user data), `tall` cell density allows for more breathing room surrounding the content.
 - If content is largely string/text-based, `short` allows for more content to be displayed within the page.
-- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
\ No newline at end of file
+- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
+
+## Multi-Selection
+
+Multi-selection enables users to select multiple rows in a table, enhancing efficiency and minimizing errors. This feature facilitates bulk operations, boosting productivity and optimizing data management.
+
+To enable multi-selection in a table:
+
+1. `Header Column / Selection` Component:
+Position it in the table's header row. This acts as a master control, enabling you to either select all rows simultaneously or deselect them.
+
+
+![Example of centered content within a table](/assets/components/table/multi-select-header.png)
+
+2. `Cell / Selection` Component:
+Integrate this into each table row. It allows for individual row selection and aligns with the Header Column component, reflecting the overall selection status of the table.
+
+![Example of centered content within a table](/assets/components/table/multi-select-cells.png)
+
+!!! Info 
+
+In the Ember component, unlike the `cell/select` figma component, achieving an indeterminate state is not possible. 
+!!!
+
+<video width="100%" controls loop>
+  <source
+    src="/assets/components/table/multi-selection-interaction-01.mp4"
+    type="video/mp4"
+  />
+</video>
+
+### Intended interaction
+
+- Selecting an individual row highlights it, indicating it is ready for actions such as editing or deletion.
+- Activating the top-level header checkbox selects every row on the current page. 
+     - This action ensures that all visible rows on the page, including those previously selected individually, are chosen collectively.
+- The 'Select all' function impacts only the rows displayed on the current page in paginated tables. In contrast, a global 'Select all' option encompasses every row across all pages (for example, selecting all 53 users in a dataset).
```

Comment:
I wouldn't use the term "Select all" here because 1) there isn't a label present that expresses this concept (even if we mean to be coupled with the Header Column / Selection), and 2) because this implies that we also provide a "Select all" function with a paginated table, which we do not.